core/domain/caching_services_test.py

```diff
@@ -174,7 +174,7 @@ def test_get_multi_correctly_retrieves_cache_elements(self):
 
     def test_partial_fetches_returns_correct_elements(self):
         """Testing that querying the cache for elements where only a subsection
-        of the queried ids exist in the cache, returns reasonable output.
+        of the queried ids exist in the cache returns reasonable output.
```

Comment:
Maybe move "returns reasonable output" to just after "querying the cache", for better readability.

Reply:
Done.